conda-env/dev.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
     - numpy=1.22.3
     - pandas=1.4.1
     - xarray=2022.3.0
+    - xesmf=0.6.2
```

Comment:
@chengzhuzhang found that `xcdat` and our conda environments break with this error:
```
File ~/miniconda3/envs/xcdat_test_dev/lib/python3.10/site-packages/xesmf/smm.py:7, in <module>
      4 import warnings
      5 from pathlib import Path
----> 7 import numba as nb
      8 import numpy as np
      9 import sparse as sps

ModuleNotFoundError: No module named 'numba'
```

@jasonb5, the conda-forge feedstock recipe for `xesmf=0.6.2` doesn't include `numba` as a dependency. This doesn't affect the PyPI release (`pangeo-xesmf`). I opened up an issue in pangeo-data/xESMF so that can fix it: https://github.com/pangeo-data/xESMF/issues/170

**In the interim, we need to include `numba` in our conda envs and https://github.com/conda-forge/xcdat-feedstock/issues/2.**

Reply:
@jasonb5 Thanks, we can wait until we do the next release to add all of the new dependencies to the recipe (https://github.com/conda-forge/xcdat-feedstock/issues/2)